src/main/java/com/google/api/codegen/transformer/PhpSurfaceNamer.java

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer;
+
+import com.google.api.codegen.CollectionConfig;
+import com.google.api.codegen.MethodConfig;
+import com.google.api.codegen.ServiceMessages;
+import com.google.api.codegen.util.Name;
+import com.google.api.codegen.util.PhpNamer;
+import com.google.api.tools.framework.model.Field;
+import com.google.api.tools.framework.model.Interface;
+import com.google.api.tools.framework.model.Method;
+import com.google.api.tools.framework.model.TypeRef;
+
+public class PhpSurfaceNamer extends SurfaceNamer {
```

Comment:
I don't think it's quite the right stage to go around making everything package-private and final that can be. I think that makes sense when toolkit becomes a library that other libraries depend on, but we're not there yet.